### ISSUE TITLE
Discord Status

### ISSUE THREAD

**Issue by OpenAndrex** [RiversideValley/Emerald#20]
### ⏱️ Before you start...

- [X] I have checked whether a similar feature request has already been reported.

### 📄 Description

Add a Discord Rich Presence for Emerald

**Comment by NoobNotFound** [comment]
yeah we can do it it's easy but I have never tried on WinUI 3 app

**Comment by NoobNotFound** [comment]
oh shit this now 3.5 years old
